### PR TITLE
解决License 按钮中svg事件冒泡问题

### PR DESCRIPTION
**Pull request by leipeizhao**
解决License 按钮中svg事件冒泡问题

### CODE REVIEW

File: app/javascript/components/tags/TagSidebar.vue [hunk 1]
```diff
@@ -241,10 +241,11 @@
   }
 
   const setActiveLicenseTag = (e) => {
-    if (activeLicenseTag.value === e.target.dataset.tag_name) {
+    const span = e.target.closest('span'); 
+    if (activeLicenseTag.value === span.dataset.tag_name) {
```
Comment (hiveer) [inline app/javascript/components/tags/TagSidebar.vue:245]:
```suggestion
    if (activeLicenseTag.value === span.textContent.trim()) {
```

File: app/javascript/components/tags/TagSidebar.vue [hunk 2]
```diff
@@ -241,10 +241,11 @@
...
       activeLicenseTag.value = ''
     } else {
-      activeLicenseTag.value = e.target.dataset.tag_name
+      activeLicenseTag.value = span.dataset.tag_name
```
Comment (hiveer) [inline app/javascript/components/tags/TagSidebar.vue:248]:
```suggestion
      activeLicenseTag.value = span.textContent.trim()
```

Comment (leipeizhao) [reply app/javascript/components/tags/TagSidebar.vue:248]: span.textContent.trim ()  这个获取的License大小写错误

**Comment by Rader** [comment]
@leipeizhao 请合并最新的代码，并更新 PR

**Comment by hiveer** [comment]
@leipeizhao Our codebase has been transferred to the Go version, please create a new MR to main branch.

**Comment by Rader** [comment]
close as no response

**Pull request closed without merging**